mypy/test/testfinegrained.py

```diff
@@ -164,7 +165,7 @@ def get_options(self, source: str, testcase: DataDrivenTestCase, build_cache: bo
         return options
 
     def run_check(self, server: Server, sources: list[BuildSource]) -> list[str]:
-        response = server.check(sources, export_types=True, is_tty=False, terminal_width=-1)
+        response = server.check(sources, export_types=False, is_tty=False, terminal_width=-1)
```

Comment:
Hmm why do we have `export_types=False` here, but true above?

Reply:
Passing it explicitly on each run will force reloading files. Putting it in options from the start is IMO cleaner.